donut.html

```diff
@@ -4,6 +4,10 @@
   <meta charset="UTF-8">
   <title>Donut Shop</title>
   <link rel="stylesheet" href="donut.css">
+<<<<<<< HEAD
+  <link href='http://fonts.googleapis.com/css?family=Eater' rel='stylesheet' type='text/css'>
+=======
+>>>>>>> 914f29507db48ce2da2bb92b6197a6be832c4cca
```

Comment:
Yeah I fixed it later :)


Sent from my Verizon Wireless 4G LTE smartphone

<div>-------- Original message --------</div><div>From: Scott Schmidt <notifications@github.com> </div><div>Date:04/14/2015  8:14 AM  (GMT-08:00) </div><div>To: TechTakeover/DonutShop <DonutShop@noreply.github.com> </div><div>Cc: Colby Wilson <colbywilson@hotmail.com> </div><div>Subject: Re: [DonutShop] Events (#5) </div><div>
</div>
> @@ -4,6 +4,10 @@
>    <meta charset="UTF-8">
>    <title>Donut Shop</title>
>    <link rel="stylesheet" href="donut.css">
> +<<<<<<< HEAD
> +  <link href='http://fonts.googleapis.com/css?family=Eater' rel='stylesheet' type='text/css'>
> +=======
> +>>>>>>> 914f29507db48ce2da2bb92b6197a6be832c4cca

Looks like you have a merge conflict here. Did you get this resolved in a later commit/push?

---
Reply to this email directly or view it on GitHub:
https://github.com/TechTakeover/DonutShop/pull/5/files#r28336198